### PR TITLE
471: update the DEVELOPER_GUIDE.md instructions for running locally

### LINKED ISSUE #471: Update the readme with latest run-env.sh and .env information
**Overview of the Feature Request**

We should mention in the docs that a new step is now required for setting up the development environment: creating a .env file from .env.example with S3 secrets.

We can also include a more friendly way to setup the environment, by simply running 
`./run-env.sh unstable` without having to worry so much about the images and container registries.

**What kind of user is the feature intended for?**
Frontend devs

**What inspired the request?**
Discussion with @pdurbin 

**What existing behavior do you want changed?**
N/A

**Any brand-new behavior do you want to add to Dataverse?**
N/A

**Any open or closed issues related to this feature request?**
N/A

### PR DESCRIPTION
**Pull request by ekraffmiller**
## What this PR does / why we need it:
Adds instructions for updating .env with S3 Credentials, and description of the 'unstable' tag.

## Which issue(s) this PR closes:

- Closes #471 

## Special notes for your reviewer:
I'm not sure what this meant from the issue description:

_We can also include a more friendly way to setup the environment, by simply running
./run-env.sh unstable without having to worry so much about the images and container registries._

So I added more descriptive text, is there more that should be done?

## Suggestions on how to test this:
Review the README and check the steps for local setup

## Does this PR introduce a user interface change? If mockups are available, please link/include them here:
No

## Is there a release notes update needed for this change?:
No

### CODE REVIEW

File: DEVELOPER_GUIDE.md [hunk 1]
```diff
@@ -210,7 +210,12 @@ As the script argument, add the name of the Dataverse image tag you want to depl
 ```bash
 # /dev-env/ directory
 
+# copy the .env.example file to .env
+# To test file upload, update the .env file with S3 credentials
+$ cp .env.example .env
+
 # Installs and runs project off latest tagged container image
+# To install the latest image from the develop branch, use the 'unstable' tag
 $ ./run-env.sh <DATAVERSE_IMAGE_TAG>
```
Comment (pdurbin) [inline DEVELOPER_GUIDE.md:219]:
```suggestion
# Installs and runs project off latest tagged container image
$ ./run-env.sh unstable

# Alternatively, you can specify a tag from https://github.com/orgs/gdcc/packages/container/package/dataverse
$ ./run-env.sh unstable <DATAVERSE_IMAGE_TAG>
```

**Review by pdurbin — COMMENTED** [review]
I added a suggestion.

File: DEVELOPER_GUIDE.md (hunk 1 above)
Comment (ekraffmiller) [reply DEVELOPER_GUIDE.md]: Thanks @pdurbin I made those changes and added info about updating the REGISTRY in .env

**Review by pdurbin — APPROVED** [review]
I didn't test this but it looks good to me!

**Comment by ofahimIQSS** [comment]
Verified and ensured text looks good without spelling errors

**Pull request merged**